src/toil/test/provisioners/aws/awsProvisionerTest.py

```diff
@@ -293,9 +322,9 @@ def _runScript(self, toilOptions):
             command.extend(toilOptions)
             self.sshUtil(command)
 
-    @integrative
-    def testAutoScaledCluster(self):
-        self._test()
+    # @integrative
```

Comment:
This was commented out while debugging to see if the restart test was the problem. I hadn't really expected a review yet.